source/documentation/rules-g.html.md.erb

```diff
@@ -0,0 +1,1208 @@
+
+If the field &lt;TRANSIT OPERATION.MRN&gt; is PRESENT in the rejected message, this Data Item shall be filled in. 
+
+<b>Technical Description</b>
+
+N/A
+
+##G0041
+
+<b>Functional Description</b>
+
+In a case when National risk is identified, by default, the value ‘X’ shall be included in &lt;RiskAnalysisIdentification.code&gt; when generated by the risk engine. However, if the sender would likely consider such information sent by the Office of Export /Office of Departure as useless for the recipient, then Office of Export /Office of Departure should use the value ‘N’.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0042
+
+<b>Functional Description</b>
+
+This D.G./D.I. is not used to report discrepancies. The Control Message always reports back D.G./D.I as at declaration message. 
+
+<b>Technical Description</b>
+
+N/A
+
+##G0045
+
+<b>Functional Description</b>
+
+The information in this Data Group/Data Item will override the information included in the CC015C (or in the latest CC013C, if any).
+
+<b>Technical Description</b>
+
+N/A
+
+##G0050
+
+<b>Functional Description</b>
+
+The Reference number shall include the ARC number or the fallback eAD reference number when the ‘Type’ of the ‘Additional reference’ is C651 or C658 respectively.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0057
+
+<b>Functional Description</b>
+
+Common Code List can be extended or restricted at national level. Purely national codes are not included in Common Domain messages.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0058
+
+<b>Functional Description</b>
+
+When /*/Consignment/HouseConsignment/ConsignmentItem/PreviousDocument/type is in SET {C651, C658} the Unique Body Reference (UBR) is required to be recorded in this field.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0061
+
+<b>Functional Description</b>
+
+The information presented in in this D.G. is related to Safety & Security and to the Binding Itinerary. In case of Binding itinerary, the information entered must include the list of codes of the countries between the Office of Departure and the Office of Destination. If more information is available about the countries visited by the means of transport since it's first place of loading until the last place of unloading, it should also be added for Safety & Security purpose only.
+
+<b>Technical Description</b>
+
+N/A
+
+##G0062
+
+<b>Functional Description</b>
+
+The rules R0506 and R0507 are applied on CC015C and CC013C to ensure that the declaration does not include unnecessary and repetitive information. They must be enforced by all NTA. Considering the possibility that one Goods Item is taken out from the declaration during the control, the message CC029C and CD001C may have different content from CC015C (or CC013C or CC170C). Consequently, those rules R0506 and R0507 shall not be strictly enforced on the Common Domain messages. Certainly not by the recipient of the CD message, likely not by the sender of the CD message.
+
```

Comment:
Two issues - 
`All NTA `does not read well. I would be inclined to avoid using an abbreviation here - or else pluralise abbreviation because of use of `all`.
- The final sentence (starts with Certainly) has the wrong tone and sounds too conversational. Rewording will require discussion.